docs/release-notes.rst

```diff
@@ -1,3 +1,199 @@
+
+For BIG-IP devices in an HA cluster, when one BIG-IP with hostname is added to F5 ACI ServiceCenter, the IP corresponding to that hostname also gets added in the side menu. The application warns the user about this, and displays a deletion prompt for retaining only the hostname version of the BIG-IP. 
+
+**Workaround:** User may click **Delete** to proceed with deletion of the additional IP version of the device added to the app. OR
+user may skip the prompt and manually delete the device which got added using IP by clicking on the delete (X) icon next to the IP of the device. Repeat the same process for the peer of this device - now both devices should be seen inside the cluster with hostname.
+
+------
+
+**Floating IP auto sync and Default Gateway auto sync will not work when hosts are added in an HA cluster using hostnames**
+
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.X**
+
+User menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB** → **Export DB**, for exporting the F5 ACI ServiceCenter database. This option is not working on APIC versions 5.0.X and higher. This is working as expected on prior versions of the APIC. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
+1. Login to the APIC server via SSH
+
+2. cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter
+
+3. Use scp, winscp or any other preferred tool to copy out the f5.db file from this location. 
+
+------
+
+**Non-default docker0 configuration on APIC may affect connection to management IP of Virtual BIG-IP**
+
+The default docker0 bridge IP has the address **172.17.0.1**. If docker0 bridge IP is updated to a non-default IP followed by APIC cluster reboot, Virtual BIG-IP login (connection to management IP) will fail from the F5 ACI ServiceCenter.
+
+**Workaround:** Change the docker0 bridge IP once more after APIC reboot to a new value. 
+
+------
+
+L2-L3 Network Management
+------------------------
+
+**Error “Invalid DN <someDn>, wrong rn prefix <somePrefix> at position X/Y“**
+
+getldevinfo.json and createbigipvlan.json APIs will show an error of the type “Invalid DN <someDn>, wrong rn prefix <somePrefix> at position X/Y“
+
+Root-cause: During VLAN creation using createbigipvlan.json API, the VLAN table in the F5 ACI ServiceCenter saves VLAN database entries. One of the fields in the VLAN table is the lifDn which is the Distinguished name of Logical Interface (in the Logical device) on APIC. If during App REST API automation, anyone creates a VLAN using createbigipvlan.json and enters invalid string in lifDn parameter of the API instead of the valid input for lifDn, the app will accept it. And on a subsequent call to getldevinfo.json or createbigipvlan.json throw the aformentioned error.
```

Comment:
anyone creates a VLAN using createbigipvlan.json and enters an invalid string...  (plus 'an')

And on a subsequent call to getldevinfo.json or createbigipvlan.json, it throws the aformentioned error.